src/main/drivers/accgyro_mpu3050.c

```diff
@@ -60,6 +62,7 @@ static uint8_t mpuLowPassFilter = MPU3050_DLPF_42HZ;
 static void mpu3050Init(void);
 static bool mpu3050Read(int16_t *gyroADC);
 static bool mpu3050ReadTemp(int16_t *tempData);
+static void checkMPU3050Interrupt(bool *gyroIsUpdated);
```

Comment:
Hmm ... pointer argument is probably good choice when called through generic interface